README.md

```diff
@@ -103,40 +103,40 @@ Keep-Alive: timeout=5
 
 ### Available options
 
-| Option          | Environment Variable      | Default value | Required |  Description  |
-| -------------   |----------------------     |----------     |:--------:|---------------|
-| unleashUrl      |`UNLEASH_URL`              | n/a           | yes      | API Url to the Unleash instance to connect to |
-| unleashApiToken | `UNLEASH_API_TOKEN`       | n/a           | yes      | API token (client) needed to connect to Unleash API. |
-| clientKeys      | `UNLEASH_PROXY_CLIENT_KEYS`     | n/a           | yes      | List of client keys that the proxy should accept. When querying the proxy, Proxy SDKs must set the request's _client keys header_ to one of these values. The default client keys header is `Authorization`. |
-| proxySecrets    | `UNLEASH_PROXY_SECRETS`   | n/a           | no      | Deprecated alias for `clientKeys`. Please use `clientKeys` instead. |
-| proxyPort       | `PORT`                    | 3000          | no       | The port where the proxy should listen. |
```

Comment:
So I can't add a suggestion on a deleted line but what about using `n/a` instead of just removing it?